src/data/reducers/assignmentTypes.test.js

```diff
@@ -1,68 +1,68 @@
 import assignmentTypes from './assignmentTypes';
-import {
-  STARTED_FETCHING_ASSIGNMENT_TYPES,
-  ERROR_FETCHING_ASSIGNMENT_TYPES,
-  GOT_ASSIGNMENT_TYPES,
-  GOT_ARE_GRADES_FROZEN,
-} from '../constants/actionTypes/assignmentTypes';
+import actions from '../actions/assignmentTypes';
 
 const initialState = {
   results: [],
   startedFetching: false,
   errorFetching: false,
 };
 
-const assignmentTypesData = ['Exam', 'Homework'];
+const testingState = {
```

Comment:
This should be an arbitrary shape (not importing the initial state, or at least adding extra fields) to appropriately test that the reducers are replacing against the passed state, and not against the initial state.